src/app/containers/Swap/utils/priceImpact.ts

```diff
@@ -0,0 +1,40 @@
+import { BigNumber } from "@ethersproject/bignumber";
+import { Zero } from "@ethersproject/constants";
+import { ethers } from "ethers";
+
+export function isHighPriceImpact(priceImpact: BigNumber): boolean {
+  // assumes that priceImpact has 18d precision
+  const negOne = BigNumber.from(10)
+    .pow(18 - 2)
```

Comment:
it seems that we are mixing the way we do our math, here you are using utility function while here https://github.com/Snowball-Finance/axial-frontend/pull/105/files#diff-158060e1d0fa9e419b7fe85ef1b8df7b3e74c8e5f214937a8232dcb193bcbf5aR297 youre not.
Please lets be consistent.